api.planx.uk/modules/lps/controller.ts

```diff
@@ -66,13 +66,7 @@ export const downloadHTMLController: DownloadHTML = async (_req, res, next) => {
   const { sessionId, email } = res.locals.parsedReq.body;
   try {
     const html = await generateHTML(sessionId, email);
-    return (
-      res
-        // Cache for 2hrs - submitted applications can't change so the data will not become stale
-        .setHeader("Cache-Control", "private, max-age=7200")
-        .header("Content-type", "text/html")
-        .send(html)
-    );
+    return res.header("Content-type", "text/html").send(html);
```

Comment:
Solving this client-side is both safer and more performant, given that we also need to control the token generation process.